lib/runner.js

```diff
@@ -235,6 +235,9 @@ Runner.prototype.fail = function(test, err) {
  */
 
 Runner.prototype.failHook = function(hook, err){
+  if(hook.parent && hook.parent.file){
```

Comment:
Small style nitpick - mind adding a space after the `if`? It'd be awesome if you could just amend or squash things to keep it 1 commit.